lib/Dancer2/CLI/Gen.pm

```diff
@@ -301,20 +303,19 @@ sub _version_check {
 
     my $latest_version = 0;
     my $resp = HTTP::Tiny->new( timeout => 5 )->get( 'https://fastapi.metacpan.org/release/Dancer2' );
-    if( $resp->{ success } ) {
-        if ( decode_json( $resp->{ content } )->{ version } =~ /(\d\.\d+)/ ) {
-            $latest_version = $1;
-        } else {
-            die "Can't understand fastapi.metacpan.org's reply.\n";
-        }
-    }
-
-    if ($latest_version gt $version) {
-        print qq{
+    if( $resp->{ success } && decode_json( $resp->{ content } )->{ version } =~ /(\d\.\d+)/ ) {
+        $latest_version = $1;
+        if ($latest_version gt $version) {
+            print qq{
 The latest stable Dancer2 release is $latest_version. You are currently using $version.
 Please check https://metacpan.org/pod/Dancer2/ for updates.
 
 };
+        }
+    } else {
+        warn "\nCouldn't determine latest version of Dancer2. Please check your internet
+connection, or use pass -x to gen to bypass this check in the future.\n\n";
```

Comment:
Terribly minor: "use -x" or "pass -x" would both make sense (the former more so, I'd say) but "use pass -x" sounds as though it's talking about running a `pass -x` command to me, rather than passing the `-x` option

Reply:
I started with use, rewrote *most* of it, and forgot to remove that! Thanks for catching... fixed!